backend/OR-Tools/OR-Tools/Solvers/OrToolsConverter.cs

```diff
@@ -0,0 +1,245 @@
+            {
+                for (int j = 0; j < locationsAmount; ++j)
+                {
+                    if (i == j) locations[i, j] = SAME_LOCATION;
+                    else locations[i, j] = NO_CROSS_BETWEEN_LOCATION;
+                }
+            }
+            return locations;
+        }
+
+        private long[,] GenerateDistancesMatrix(FileInput input)
+        {
+            int locationsAmount = input.Locations.Count();
+            long[,] distances = GenerateLocationMatrix(locationsAmount);
+            foreach (Distances distance in input.Distances)
+            {
+                distances[distance.From - 1, distance.To - 1] = distance.Distance;
+                distances[distance.To - 1, distance.From - 1] = distance.Distance;
+            }
+            return distances;
+        }
+
+        private long[,] GenerateDurationsMatrix(FileInput input)
+        {
+            int locationsAmount = input.Locations.Count();
+            long[,] durations = GenerateLocationMatrix(locationsAmount);
+            foreach (Distances distance in input.Distances)
+            {
+                durations[distance.From - 1, distance.To - 1] = distance.Duration;
+                durations[distance.To - 1, distance.From - 1] = distance.Duration;
+            }
+            return durations;
+        }
+        #endregion
+
+        #region ToOutput
+        public FileOutput ConvertToFileOutput(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
+        {
+            return new FileOutput()
+            {
+                DroppedLocation = GetDroppedLocations(input, manager, routing, solution),
+                Itineraries = GetItineraries(input, manager, routing, solution),
+                Summaries = GetSummaries(input, manager, routing, solution),
+                Totals = GetTotals(input, manager, routing, solution)
+            };
+        }
+
+        private IList<Dropped> GetDroppedLocations(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
+        {
+            IDictionary<int, string> locationsNames = input.Locations.ToDictionary(k => k.Id - 1, v => v.Name);
+            List<Dropped> droppedLocations = new List<Dropped>();
+            for (int index = 0; index < routing.Size(); ++index)
+            {
+                if (routing.IsStart(index) || routing.IsEnd(index))
+                {
+                    continue;
+                }
+
+                if (solution.Value(routing.NextVar(index)) == index)
+                {
+                    int node = manager.IndexToNode(index);
+                    droppedLocations.Add(new Dropped { LocationName = locationsNames[node] });
+                }
+            }
+            return droppedLocations;
+        }
+
+        private IList<Itineraries> GetItineraries(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'manager'.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)